README.md

```diff
@@ -45,6 +45,13 @@ Instead, it is a **support system** for Hiero SDK contributors, meant to lower t
 
 This project is maintained by the Hiero organization. For questions or support, open an issue or reach out to a [maintainer](MAINTAINERS.md).
 
+## Hiero SDK Call 
+
+We meet fortnightly on Monday's to collaborate on Hiero's SDK. Minutes are posted under /minutes.
```

Comment:
```suggestion
We meet fortnightly on Monday's to collaborate on Hiero's SDK.
```

Maybe instead of referencing `/minutes` here, you can add it to the table above in the **What's in this Repo?** section?